### PR TITLE
add permission property in field properties documentation

### PR DESCRIPTION
**Pull request by ValJed**

### CODE REVIEW

File: docs/reference/field-properties/README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ Each [Schema Field Type](/reference/field-types) in Apostrophe can take a number
 |help | `string` | | Help text for the field that will appear with the field's label | universal | |
 |htmlHelp | `string` | | Help text with support for HTML markup | universal | |
 |contextual | `boolean` | false | If true, it will prevent the field from appearing in the dialog box for a widget | universal | |
+|permission | `string` | | Specify the permission level needed to edit this field | universal | |
```
Comment (abea) [inline docs/reference/field-properties/README.md:17]: Please link to this page from the text "the permission level needed": `/core-concepts/users-and-permissions/managing-access-control.md`

**Review by abea — CHANGES_REQUESTED** [review]

File: docs/reference/field-properties/README.md (hunk 1 above)
Comment (ValJed) [reply docs/reference/field-properties/README.md]: Yes, done !

File: docs/reference/field-properties/README.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ Each [Schema Field Type](/reference/field-types) in Apostrophe can take a number
 |help | `string` | | Help text for the field that will appear with the field's label | universal | |
 |htmlHelp | `string` | | Help text with support for HTML markup | universal | |
 |contextual | `boolean` | false | If true, it will prevent the field from appearing in the dialog box for a widget | universal | |
+|permission | `string` | | Specify [the permission level needed](../../core-concepts/users-and-permissions/managing-access-control.md) to edit this field | universal | |
```
Comment (abea) [inline docs/reference/field-properties/README.md:17]: `/core-concepts` is a top level section, so you can remove the `../../` at the beginning. So, it should simply be `/core-concepts/users-and-permissions/managing-access-control.md`.

**Review by abea — CHANGES_REQUESTED** [review]

File: docs/reference/field-properties/README.md (hunk 2 above)
Comment (ValJed) [reply docs/reference/field-properties/README.md:17]: Done

**Review by abea — APPROVED** [review]

**Pull request merged**